element_electrode_localization/coordinate_framework.py

```diff
@@ -56,13 +71,124 @@ class Voxel(dj.Part):
         -> CCF.Voxel
         """
 
+    @classmethod
+    def retrieve_acronym(self, acronym):
```

Comment:
a classmethod should not take a `self`. Typically the first argument is called `cls`.